### PR TITLE
Embedding Images and Video Project (Youtube Clone)

### PR DESCRIPTION
**Pull request by VanessaAoki**
We made a clone of the youtube player page, it has a functional video, buttons, and interactive icons. The page works best on a 1060x800 resolution and using the firefox browser, assets might look out of place using other browsers or resolutions. It has mock suggestions showing video name and views.

The focus of this branch was to make everything fit in their right places, along with a functional youtube video player.

### CODE REVIEW

File: styled.css [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+}
+.recommended-video-box5 {
+    padding-top: 120px;
+    width: 260px;
+    padding-left: 30px;
+    box-shadow:#a9a9a9;
+    background-color: #ffffff;
+}
+p {
+    margin: 0px;
```
Comment (arslanbisharat) [inline styled.css]: Values of 0 shouldn't have units specified.

File: styled.css [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+}
+.avatar {
+    margin-top: 20px;
+    height: 30px;
+    float: right;
+    padding-right: 10px;
+}
+.video {
+    display: inline-block;
+    padding-left: 0px;
```
Comment (arslanbisharat) [inline styled.css]: Values of 0 shouldn't have units specified.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href="/styled.css">
+        <title>youtube mockpage</title>
+    </head>
+    <body>
+    <div id="wrapper">
+        <header>
+            <img class="logo" src="/images/logo.png">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute

File: index.html [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+            <input class="search-bar">
+            <button class="search-icon">
+                <img class="search-img" src="/images/search-icon.png">
+            </button>
+            <img class="avatar" src="/images/avatar.png">
+            <img class="menu" src="images/menu.png">
+            <img class="upload" src="/images/upload.png">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute

File: index.html [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+        <link rel="stylesheet" type="text/css" href="/styled.css">
+        <title>youtube mockpage</title>
+    </head>
+    <body>
+    <div id="wrapper">
+        <header>
+            <img class="logo" src="/images/logo.png">
+            <input class="search-bar">
+            <button class="search-icon">
+                <img class="search-img" src="/images/search-icon.png">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute

File: index.html [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+            <img class="upload" src="/images/upload.png">
+        </header>
+            <div class="video">
+                <iframe width="630" height="315" src="https://www.youtube.com/embed/V74l_zS1x8E" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
+                </iframe>
+                <div class="video-text">
+                <p class="video-title">
+                    Contributing to Open Source Part I: The Easy Way 
+                </p>
+                    <img class="avatar2" src="/images/avatar2.png">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute

File: index.html [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+                    <p class="avatar-name">The Odin Project</p> 
+                    <p class="view-number">
+                        50,016 views    
+                    </p>
+                    <img class="share-button" src="/images/share.png">
+                    <img class="subscribe" src="/images/subscribe.png">
+                    <img class="add" src="/images/Add.png">
+                    <img class="more" src="images/More.png">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute

File: index.html [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+                    <div class="like-dislike">
+                        <img src="/images/like.png">
+                        <img src="/images/dislike.png">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute

File: index.html [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+                    </div>
+                    </div>
+            </div>
+
+            <div class="sidebar">
+                <p class="upnext">Up Next</p> 
+                <div class="recommended-video-box1">
+                    <img class="recommended-video" src="/sidebar/Contributiong to Open Source PartII: The Real Way.jpeg">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute

File: index.html [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+                    <div class="recommended-text">    
+                        <span class="recommended-title">Contributing to Open Source Part II: The Real Way</span>
+                        <p>The Odin Project</p>
+                        <p>29,985 views</p>
+                    </div>
+                </div>
+                <div class="recommended-video-box2">
+                    <img class="recommended-video" src="/sidebar/Contributiong to Open Source PartII: The Real Way.jpeg">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute

File: index.html [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+                    <div class="recommended-text">    
+                        <span class="recommended-title">Contributing to Open Source Part II: The Real Way</span>
+                        <p>The Odin Project</p>
+                        <p>29,985 views</p>
+                    </div>
+                </div>
+                <div class="recommended-video-box3">
+                    <img class="recommended-video" src="/sidebar/Contributiong to Open Source PartII: The Real Way.jpeg">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute

File: index.html [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+                    <div class="recommended-text">    
+                        <span class="recommended-title">Contributing to Open Source Part II: The Real Way</span>
+                        <p>The Odin Project</p>
+                        <p>29,985 views</p>
+                    </div>
+                </div>
+                <div class="recommended-video-box4">
+                    <img class="recommended-video" src="/sidebar/Contributiong to Open Source PartII: The Real Way.jpeg">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute

File: index.html [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+                    <div class="recommended-text">    
+                        <span class="recommended-title">Contributing to Open Source Part II: The Real Way</span>
+                        <p>The Odin Project</p>
+                        <p>29,985 views</p>
+                    </div>
+                </div>
+                <div class="recommended-video-box5">
+                    <img class="recommended-video" src="/sidebar/Contributiong to Open Source PartII: The Real Way.jpeg">
```
Comment (arslanbisharat) [inline index.html]: An img element must have an alt attribute

File: index.html [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+                    <div class="recommended-text">    
+                        <span class="recommended-title">Contributing to Open Source Part II: The Real Way</span>
+                        <p>The Odin Project</p>
+                        <p>29,985 views</p>
+                    </div>
+                </div>
+                </div>
+            </div>
+    </div>
```
Comment (arslanbisharat) [inline index.html]: Stray end tag div

File: index.html (hunk 3 above)
Comment (arslanbisharat) [inline index.html]: path should be src="./images/logo.png"

File: index.html (hunk 5 above)
Comment (arslanbisharat) [reply index.html]: the path should be `./images/search-icon.png`

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**


- [x]  Please make your PR descriptive. Please add more details to it. Please refer to [link ](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642) from the pathway to learn more. 

- [x] I cannot see anything from your project because  Your project looks like this. 

**Clone**

chrome-capture (14)

The problem is that you have to use the proper path in the `src` attribute of your HTML tag. Please use `./specific path`.

- [x] I have given some suggestions below. Please have a look at the inline-comments and work on them. 

- [x] Please make your Readme more descriptive by 
  - adding the proper links to your social media accounts and 
  - a live link to your project. 
  - You can add details about your project and also you can add how to clone this project and acknowledgments. This [blog ](https://blog.bitsrc.io/how-to-write-beautiful-and-meaningful-readme-md-for-your-next-project-897045e3f991?gi=bddbb8c7067e)might help you. 

- [x] Please move images from the sidebar folder to the images folder and then move the images folder and styled.css to one folder so that your repo looks clean. 


**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

File: README.md [hunk 15]
```diff
@@ -1,2 +1,62 @@
-# youtube-project
-Project: Embedding Images and Video
+# Youtube Clone Page
+
+> This project consists of building an HTML document that matches the appearance of youtube video player page.
```
Comment (arslanbisharat) [inline README.md]: **Change it to:** This project consists of building an HTML document that matches the appearance of the [youtube video player page](http://archive.vn/Bss88).

File: README.md [hunk 16]
```diff
@@ -1,2 +1,62 @@
...
+
+![screenshot](./images/Screenshot.png)
+
+Additional description about the project and its features.
```
Comment (arslanbisharat) [inline README.md]: Please remove this line or add description and features

File: README.md [hunk 17]
```diff
@@ -1,2 +1,62 @@
...
+
+### Prerequisites
+
+- Live Server plugin for Visual Studio Code 
+
+### Setup
+
+1- Clone the repository
+```
+git clone https://github.com/elberthcorniell/Youtube-Clone.git
```
Comment (arslanbisharat) [inline README.md]: **Please change it to** https://github.com/VanessaAoki/youtube-project

File: README.md [hunk 18]
```diff
@@ -1,2 +1,62 @@
...
+
+## Authors
+
+👤 **Vanessa Aoki**
+
+- GitHub: [@VanessaAoki](https://github.com/VanessaAoki)
+
+👤 **Angela Cuahutle**
+
+- GitHub: [@angelacuahutle](https://github.com/angelacuahutle)
```
Comment (arslanbisharat) [inline README.md]: Please add more social links like Linkedin Twitter etc.

File: README.md [hunk 19]
```diff
@@ -1,2 +1,62 @@
...
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](issues/).
```
Comment (arslanbisharat) [inline README.md]: Please add a proper issue page.

File: README.md [hunk 20]
```diff
@@ -1,2 +1,62 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](https://github.com/elberthcorniell/Youtube-Clone/blob/video-frame/LICENSE.md) licensed.
```
Comment (arslanbisharat) [inline README.md]: Please add your own License

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

- [x]  Please make your PR descriptive. Please add more details to it. Please refer to [link ](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642) from the pathway to learn more.
```

We made a clone of the youtube player page, it has a functional video, buttons, and interactive icons. The page works best on a 1366x768 resolution and using the firefox browser, assets might look out of place using other browsers or resolutions. It has mock suggestions showing video name and views, and a mock section for commentaries.

The focus of this branch was to make everything fit in their right places, along with a functional youtube video player.
```

- [x] Please make your Readme descriptive without mistakes. I have added some suggestions below. 

- [ ] Please remove spacing from the side of your page.

**Clone**

chrome-capture (15)

- [x] Hamburger is missing. 

**Clone**

chrome-capture (16)

- [x] There should three icons and one avatar. But you have two icons and one avatar. 

chrome-capture (17)

- [ ] [Optional] No item in this section is clickable. Please make the subscribe button and the rest of the items' proper link. 

chrome-capture (18)

- [x] [Optional] Autoplay button is missing here. 

**Clone**

chrome-capture (19)

- [ ] [Optioanl] It's good to have Comment section too. 


**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Comment by mimipeshy** [comment]
**STATUS : APPROVED :partying_face:  :tada:** 

Congratulations @VanessaAoki and @angelacuahutle  on working so hard on this project. It is approved !!!! I wish you all the best ahead in your coding journey. Cheers

Regards
Peris Ndanu

**Pull request merged**